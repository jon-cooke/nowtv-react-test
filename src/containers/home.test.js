import React from 'react';
import { shallow } from 'enzyme';

import Home from './home';

it('renders correctly', () => {
  const props = {
    messages: [],
    loadMessages: () => {},
  };
  const tree = shallow(<Home.WrappedComponent {...props} />);
  expect(tree).toMatchSnapshot();
});

it('renders a single message correctly', () => {
  const props = {
    messages: [
      {
        id: 'cd445e6d-e514-424f-ba8f-16ec842002c6',
      },
    ],
    loadMessages: () => {},
  };
  const tree = shallow(<Home.WrappedComponent {...props} />);
  expect(tree).toMatchSnapshot();
});
