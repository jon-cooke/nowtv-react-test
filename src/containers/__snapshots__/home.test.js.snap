// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a single message correctly 1`] = `
<div>
  <Message
    id="cd445e6d-e514-424f-ba8f-16ec842002c6"
    key="cd445e6d-e514-424f-ba8f-16ec842002c6"
  />
</div>
`;

exports[`renders correctly 1`] = `<div />`;
